WD-Firmware/demo/build/SConstruct

```diff
@@ -132,14 +137,20 @@ Env['BUILDERS']['Size'] = Builder(action = utils.fnProduceSectionsSize)
 Env['PUBLIC_DEF'] = objDemo.public_defs
 Env['TARGET_LIBS'] += ['c', 'gcc']
 
+#prepare all Sconscripts for build
+objDemo.listSconscripts.append(strBsp+ Env['TARGET_BOARD']) 
+
 Env["Scan"] = True
-SConscript('SConscript_bsp_'+ Env['TARGET_BOARD'], exports='Env')
 for strSconscript in objDemo.listSconscripts:
   SConscript('SConscript_%s' % strSconscript, exports='Env')

```

Comment:
make a string variable at the beginning of the file
strSconscriptPrefix = "SConscript_%s"

Reply:
done but without the %s (have to be const)
strSconscriptPrefix = 'SConscript_'